test/OperandCapabilities.cpp

```diff
@@ -38,23 +47,17 @@ struct EnumCapabilityCase {
 };
 
 // Test fixture for testing EnumCapabilityCases.
-template <spv_target_env env>
-class EnumCapabilityTest : public ::testing::TestWithParam<EnumCapabilityCase> {
- protected:
-  const spv_target_env env_ = env;  // Target environment to use in tests.
-};
-
-using EnumCapabilityTestV10 = EnumCapabilityTest<SPV_ENV_UNIVERSAL_1_0>;
-using EnumCapabilityTestV11 = EnumCapabilityTest<SPV_ENV_UNIVERSAL_1_1>;
+using EnumCapabilityTest =
+    TestWithParam<tuple<spv_target_env, EnumCapabilityCase>>;
 
-TEST_P(EnumCapabilityTestV10, Sample) {
```

Comment:
This was a test case only in the `V10` fixture, leaving the `V11` fixture empty.